src/style/shared/colors-medium.less

```diff
@@ -59,6 +60,7 @@
 
 @focus-highlight: #3691ff;
 @text-highlight: #1B8BE9;
+@text-focus: #1473e6;
```

Comment:
For code cleanliness sake, can you make sure hex letters are uppercase like the examples above?